bridgetown-core/lib/bridgetown-core/configurations/tailwindcss.rb

```diff
@@ -21,9 +21,14 @@
 
 copy_file "#{TEMPLATE_PATH}/postcss.config.js", "postcss.config.js", force: true
 
-prepend_to_file "frontend/styles/index.css",
-                File.read("#{TEMPLATE_PATH}/css_imports.css")
-
 run "bundle exec bridgetown configure purgecss"
 
+if File.exist?("frontend/styles/index.css")
```

Comment:
Just guarding against a case where `index.css` doesn't exist and printing out what the user needs to include in their CSS.

Unlikely to happen but could be the case if someone created a site using sass and then converted to postcss. We don't update the CSS/JS setup when enabling postcss on an existing site as it'd be too complicated.